AppController/lib/app_dashboard.rb

```diff
@@ -65,7 +63,7 @@ def self.start(public_ip, private_ip, persistent_storage, secret)
 
     Djinn.log_debug("Done setting dashboard.")
 
-    return true
+    return app_location
```

Comment:
Could you update the returns section in the doc string to be the app location instead of a boolean.